### PR TITLE
docker-compose + blog post

### PR DESCRIPTION
**Pull request by beforan**
This PR replaces #18 

It includes a `docker-compose.yml` file to make getting started easier, along with guidance in the README.

It also includes a blog post by me :)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -78,7 +77,6 @@ DEPENDENCIES
   jekyll-feed (~> 0.6)
   minima (~> 2.0)
   tzinfo-data
-  wdm (~> 0.1.0)
```
Comment (beforan) [inline Gemfile.lock]: @jamietanna @AnnaDodson I confirmed what we thought on this in #18 

If i run `bundle install` in windows it adds `wdm` and if i run it in WSL or using the docker version, it removes `wdm`. repeatably.

So I don't think we can do anything about that at the mo.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,45 @@
...
+You'll need to install Docker on your operating system.
+
+- For **Mac** and **Windows**, install [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- For **Linux**, install [Docker Engine Community Edition](https://store.docker.com/search?type=edition&offering=community)
+
+:information_source: Ensure when you install that you are using **Linux containers**, not Windows ones (even if you're installing on Windows).
+
+Once docker is installed, simply go to the root of the project folder in a terminal window, and run:
+
+`docker-compose up`
```
Comment (AnnaDodson) [inline README.md]: I have to install docker-compose first on Ubuntu - it's not included in the docker CE install by default. Is it there already for Windows? Might be worth adding a line to check it's installed and install it if it's not.

https://docs.docker.com/compose/install/

File: Gemfile.lock (hunk 1 above)
Comment (AnnaDodson) [reply Gemfile.lock]: This is ok for me, I can toggle between running locally or in docker and the Gemfile sorts itself out. We could potentially add a troubleshooting section in the README if this is a problem for some people.

File: README.md (hunk 2 above)
Comment (beforan) [reply README.md]: Good shout, yeah the Windows "Docker Desktop" bundle includes it, didn't realise it didn't on Linux

Comment (AnnaDodson) [reply README.md]: Great! Thanks. Also if you do a pull from master before committing, it should resolve the merge conflict in the author's file. Thanks

Comment (beforan) [reply README.md]: merged your upstream master, and added the docker-compose note 👍

**Pull request merged**